camel-route-support/src/main/java/io/kaoto/backend/api/service/step/parser/camelroute/CamelRouteStepParserService.java

```diff
@@ -40,19 +40,29 @@ public String description() {
 
     @Override
     public ParseResult<Step> deepParse(final String input) {
+        // Right now we discard any flow that is not the first
+        return getParsedFlows(input).get(0);
+    }
+
+    @Override
+    public List<ParseResult<Step>> getParsedFlows(final String input) {
```

Comment:
@Delawen, Initially, I was returning an array of Results, but it seems like the `Integration` class already supports multiple flows, I'm not sure about the `CamelRoute` class though.

That being said, I'm wondering if a better way would be to update the `ParseResult` class to support multiple flows instead. What do you think?